rfcs/syntax-default-type-alias-type-parameters.md

```diff
@@ -49,7 +49,7 @@ type A<T, U = V, V = T> = ... -- not allowed
 Default type parameter values are also allowed for type packs:
 ```lua
 type A<T, U... = ...string> -- ok, variadic type pack
-type C<T, U... = string>    -- ok, type pack with one element
+type C<T, U... = (string)>  -- ok, type pack with one element
```

Comment:
```suggestion
type C<T, U... = (string)>  -- ok, type pack with one element
type C<T, U... = (string,number)>  -- ok, type pack with two elements
type C<T, U... = ()>  -- ok, type pack with no elements
```

Reply:
Added this one and an extra one for explicit variadic type pack.
Also adjusted the names to be different (not a requirement for an example but it was like that before)